package.json

```diff
@@ -8,13 +8,14 @@
     "@wdio/mocha-framework": "^7.16.15",
     "@wdio/spec-reporter": "^7.16.14",
     "@wdio/static-server-service": "^7.16.14",
-    "autoprefixer": "^10.4.2",
+    "autoprefixer": "^10.4.16",
```

Comment:
Please revert these changes. I'm going to do the dependency updates on the upstream.

Reply:
Still not fixed.

Reply:
sure on it.